lte/gateway/c/core/oai/tasks/sctp/sctpd_downlink_client.cpp

```diff
@@ -110,8 +109,10 @@ constexpr useconds_t max_backoff_usecs = 1000000;  // 1 sec
 std::unique_ptr<SctpdDownlinkClient> client = nullptr;
 
 int init_sctpd_downlink_client(bool force_restart) {
-  auto channel =
-      grpc::CreateChannel(DOWNSTREAM_SOCK, grpc::InsecureChannelCredentials());
+  std::string downstream_sctp_sock(
+      bstr2cstr(mme_config.sctp_config.downstream_sctp_sock, '\0'));
```

Comment:
this leaks memory as bstr2cstr will allocate a new memory while string object will have its own copy of it. we should use `bdata(mme_config.sctp_config.downstream_sctp_sock)` instead.